### PR TITLE
Added local ORM classes for mlwh database.

### PR DESCRIPTION
**Pull request by mgcam**
Copied a few classes we need from the external
ml_warehouse package to lang_qc.db.mlwh_schema.
Reduced the number of imports to the types that
are actually needed. Reduced the number of mapped
columns for the study and sample tables to columns we are or might be using.

### CODE REVIEW

File: lang_qc/db/mlwh_schema.py [hunk 1]
```diff
@@ -0,0 +1,548 @@
...
+from sqlalchemy.dialects.mysql import CHAR as mysqlCHAR
+from sqlalchemy.dialects.mysql import FLOAT as mysqlFLOAT
+from sqlalchemy.dialects.mysql import INTEGER as mysqlINTEGER
+from sqlalchemy.dialects.mysql import SMALLINT as mysqlSMALLINT
+from sqlalchemy.dialects.mysql import TINYINT as mysqlTINYINT
+from sqlalchemy.dialects.mysql import VARCHAR as mysqlVARCHAR
+from sqlalchemy.orm import declarative_base, relationship
+
+Base = declarative_base()
+# metadata = Base.metadata
```
Comment (nerdstrike) [inline lang_qc/db/mlwh_schema.py:31]: Delete

File: tests/fixtures/utils.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
-from ml_warehouse.schema import (
+from sqlalchemy import delete, text
```
Comment (nerdstrike) [inline tests/fixtures/utils.py:1]: `text` seems redundant.

**Review by nerdstrike — CHANGES_REQUESTED** [review]
Looks ok, I've caught a couple of minor details.

File: lang_qc/db/mlwh_schema.py (hunk 1 above)
Comment (mgcam) [reply lang_qc/db/mlwh_schema.py]: Thanks,  deleted in https://github.com/wtsi-npg/npg_langqc/pull/138/commits/9d48fd9808fcd4e23dddc5c6d4c6a6e5385b2979

File: tests/fixtures/utils.py (hunk 2 above)
Comment (mgcam) [reply tests/fixtures/utils.py]: Thanks, removed in https://github.com/wtsi-npg/npg_langqc/pull/138/commits/9d48fd9808fcd4e23dddc5c6d4c6a6e5385b2979

**Review by nerdstrike — APPROVED** [review]

**Pull request merged**